src/glass/src/app/pages/users-page/users-page.component.scss

```diff
@@ -7,7 +7,7 @@
     margin-bottom: 8px;
 
     button {
-      @extend .glass-color-theme-primary;
+      @extend .btn-outline-default;
```

Comment:
Using a different style here is by intention. On this page the `Add` button has a direct impact to the content displayed in the the table. The outlined button on the host page does not have any impact to the table content, thus it has no color.

Reply:
Ah... I see, thanks :+1: